### ISSUE TITLE
Could there be a Setup and Teadown phase?

### ISSUE THREAD

**Issue by aanzorena** [ddosify/ddosify#222]
In my case, I have a Test Plan that only has to login into a website just once, because if it does so every time the test runs, the previous Bearer Token result is invalidated, and the test may not complete successfully. It would be nice to have the possibility to add a Setup and Teardown stage, where you can configure steps to be run only once.

Thanks in advance.

Regards

**Comment by fatihbaltaci** [comment]
Hello @aanzorena. This feature is on our roadmap. I'll update this issue when this feature is ready.

**Comment by KshitijBharde** [comment]
Hi @fatihbaltaci, can I take up this issue?

**Comment by fatihbaltaci** [comment]
Hello @KshitijBharde, yes! If you need any assistance, please let me know. Thanks.

**Comment by KshitijBharde** [comment]
Sure, @fatihbaltaci will let you know.

**Comment by KshitijBharde** [comment]
Is there a discord/slack channel where I can discuss this with the team?

**Comment by fatihbaltaci** [comment]
Yes @KshitijBharde, you can join our Discord Server: https://discord.com/invite/9KdnrSUZQg

**Comment by fatihbaltaci** [comment]
Hello @KshitijBharde, is there any progress for this issue? Thanks.
